panel/models/card.ts

```diff
@@ -106,11 +108,11 @@ export class CardView extends ColumnView {
       header.r_after_render()
     }
 
-    if (this.model.collapsed) {
-      return
-    }
-
     for (const child_view of this.child_views.slice(1)) {
+      if (this.initially_collapsed) {
```

Comment:
Not sure I entirely follow this. Won't this just set `visible=False` on the first child?

Also I forget how things work, but do we run `render()` every time the children are updated?

Reply:
> Won't this just set visible=False on the first child?

Ah yes, I think you are right! So then something else should be done 🤔 

> do we run render() every time the children are updated?

Yep, the cards widget seems like works like that (at least the `update_children` definition override is calling `render` from what I can see):

https://github.com/holoviz/panel/blob/42b09569a517a428f34d6ce0b8f01143246c8f84/panel/models/card.ts#L120-L124

Reply:
Latest commit makes the change mentioned above. Seems like things are passing with it but if you think of a different approach or something else is missing let me know!